src/controllers/list.js

```diff
@@ -791,21 +791,37 @@ class ItemsView {
 
                 const itemType = item ? item.Type : null;
 
-                if (itemType === 'MusicGenre' || params.type !== 'Programs' && itemType !== 'Channel') {
+                if ((itemType === 'MusicGenre' || params.type !== 'Programs' && itemType !== 'Channel')
+                    // Folder, Playlist views
+                    && itemType !== 'UserView'
+                    // Only Photo (homevideos) CollectionFolders are supported
+                    && !(itemType === 'CollectionFolder' && item?.CollectionType !== 'homevideos')
```

Comment:
Wouldn't that make it easier to perceive?
```suggestion
                    && (itemType !== 'CollectionFolder' || item?.CollectionType === 'homevideos')
```

Reply:
It makes it harder to understand for me, but that could just be me. :wink: